web-ui/src/main/resources/catalog/views/default/templates/home.html

```diff
@@ -139,7 +139,7 @@ <h1 class="col-md-12">
               data-ng-model="homeFacet.key"
               data-ng-value="facetKey"
             />
-            <span data-translate="">{{::('facet-' + facetKey) | facetKeyTranslator}}</span
+            <span data-translate="">{{::getFacetLabel(agg) ||  (('facet-' + facetKey) | facetKeyTranslator)}}</span
```

Comment:
I think it requires to be updated line 126 also, no?

https://github.com/geonetwork/core-geonetwork/pull/8536/files#diff-3a24b98db744cfdd8abe9968f0647c4aa21763290238175f748910529350def4L126

Also probably these other 2 files:

- https://github.com/geonetwork/core-geonetwork/blob/64f9efc758ca66466ce57c9f3430cc8f100fe4e1/web-ui/src/main/resources/catalog/components/metadataactions/partials/recordsFilters.html#L5

- https://github.com/geonetwork/core-geonetwork/blob/64f9efc758ca66466ce57c9f3430cc8f100fe4e1/web-ui/src/main/resources/catalog/components/search/searchfiltertag/partials/searchFilterTagsTemplate.html#L24

Reply:
Indeed, in Sextant those components are not used and it was not done there. Updated.